### PR TITLE
bsp: Use esp_lvgl_port for esp-box bsp (BSP-242)

### LINKED ISSUE #106: Use esp_lvgl_port as a dependency for the boards with LCD panel (BSP-238)
Since we already have [esp_lvgl_port](https://components.espressif.com/components/espressif/esp_lvgl_port), it would be better to use it in the BSPs which also have an LVGL port layer.

### PR DESCRIPTION
**Pull request by wujiangang**
Partially close #106, since the BSP for esp-box is needed at the first stage. If some modifications are needed, please update and commit directly.

### CODE REVIEW

File: esp-box/esp-box.c [hunk 1]
```diff
@@ -413,14 +292,15 @@ esp_err_t bsp_display_backlight_on(void)
...
     BSP_ERROR_CHECK_RETURN_NULL(bsp_display_brightness_init());
-    lv_disp_t *disp = lvgl_port_display_init();
-    BSP_ERROR_CHECK_RETURN_NULL(lvgl_port_indev_init());
-    BSP_ERROR_CHECK_RETURN_NULL(lvgl_port_tick_init());
-    lvgl_mux = xSemaphoreCreateMutex();
-    BSP_NULL_CHECK(lvgl_mux, NULL);
-    xTaskCreate(lvgl_port_task, "LVGL task", 4096, NULL, CONFIG_BSP_DISPLAY_LVGL_TASK_PRIORITY, NULL);
+
+    disp = bsp_display_lcd_init();
+
```
Comment (espzav) [inline esp-box/esp-box.c:301]: Check if `disp != NULL` before call the next function.

File: esp-box/esp-box.c [hunk 2]
```diff
@@ -413,14 +292,15 @@ esp_err_t bsp_display_backlight_on(void)
...
+    bsp_display_indev_init(disp);
```
Comment (espzav) [inline esp-box/esp-box.c:302]: Return `NULL`, when this function fails (returns `NULL`).

**Review by espzav — COMMENTED** [review]
Thank you for modify this BSP for using LVGL port! Great work! I left some comments and please don't forget increase the version of this component.

**Review by espzav — APPROVED** [review]
Thank you. LGTM

**Review by espzav — CHANGES_REQUESTED** [review]
Please, increase the component version in idf_component.yml

**Comment by wujiangang** [comment]
> Please, increase the component version in idf_component.yml

I want to increase the version after #101 and #113, or each PR needs to increase a minor version?

**Comment by espzav** [comment]
> I want to increase the version after #101 and #113, or each PR needs to increase a minor version?

If the version is not increased, the component will not be updated in IDF Component Registry. It's recommended increase version after each change (each PR). Some PR can take more time.

**Review by espzav — APPROVED** [review]
Thank you. LGTM

**Pull request merged**